core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -539,6 +543,176 @@ class KafkaZkClient(zooKeeperClient: ZooKeeperClient, isSecure: Boolean) extends
     retryRequestsUntilConnected(deleteRequests)
   }
 
+  //Acl management methods
+
+  /**
+   * Creates the required zk nodes for Acl storage
+   */
+  def createAclPaths(): Unit = {
```

Comment:
Could we add a unit test for each new method in KafkaZkClientTest?

Reply:
Added unit tests to acl related methods. added couple of methods related to config management.  I will add tests for these methods as part of KAFKA-5646/DynamicConfig